certbot-ci/windows_installer_integration_tests/conftest.py

```diff
@@ -22,3 +24,26 @@ def pytest_addoption(parser):
                                           'nsis', 'certbot-beta-installer-win32.exe'),
                      help='set the path of the windows installer to use, default to '
                           'CERTBOT_ROOT_PATH\\windows-installer\\build\\nsis\\certbot-beta-installer-win32.exe')
+    parser.addoption('--confirm', action='store_true',
```

Comment:
nit: Especially if we change the behavior of the flag, what do you think about a name like `--allow-persistent-changes`?

Reply:
I was thinking of a `--confirm` flag as "say yes to whatever interactive question you would encounter", quite similarly to `-y` with apt.

But if you prefer a flag for each interactive behavior, it is your call. Still prefer that ?

Reply:
I don't care that much, but while our only use of this flag is with the behavior here, I personally prefer a flag that tries to clearly explain its use. If we get multiple of these flags in the future and people are wanting to run these tests outside of CI, I would maybe prefer a flag like `--force-yes` instead of or in addition to flags specifically for each warning.

Reply:
Ok, done.